mam/v2/wots/wots.c

```diff
@@ -13,8 +13,10 @@
 \brief MAM2 WOTS layer.
 */
 
+#include <malloc.h>
```

Comment:
If you want to call `malloc(3)` and/or `free(3)`, the header `<stdlib.h>` should be included rather than `<malloc.h>`.